src/tests/ftest/control/daos_server_helper.py

```diff
@@ -56,37 +56,31 @@ def test_daos_server_helper_format(self):
         # Prep server for format, run command under non-root user
         # Note: This will just report the presence of PMem namespaces if the NVDIMMs are already
         #       configured in AppDirect interleaved mode and namespaces have been created.
-        self.log.info("Performing SCM storage prepare")
-        try:
-            self.server_managers[0].prepare_storage(user, True, False)
-        except ServerFailed as error:
-            self.fail("(1)Failed preparing SCM as user {}: {}".format(user, error))
```

Comment:
Does this mean we aren't running `daos_server scm reset`?

Reply:
We are running daos_server scm prepare, but not running the unsupported --target-user option, which been removed.